markdown/usage/nf_core_tutorial_using.md

```diff
@@ -0,0 +1,261 @@
+
+```bash
+nextflow run nf-core/rnaseq -profile test,docker
+nextflow run nf-core/rnaseq -profile singularity,debug
+```
+
+Note that the order in which config profiles are specified matters.
+Their priority increases from left to right.
+
+### Running pipelines with test data
+
+The `test` config profile is a bit of a special case.
+Whereas all other config profiles tell Nextflow how to run on different computational systems, the `test` profile configures each `nf-core` pipeline to run without _any_ other command line flags.
+It specifies URLs for test data and all required parameters.
+Because of this, you can test any _nf-core_ pipeline with the following command:
+
+```bash
+nextflow run nf-core/PIPELINE -profile test
+```
+
+Note that you will typically still need to combine this with a configuration profile for your system - _e.g._ `-profile test,docker`.
+Running with the test profile is a great way to confirm that you have Nextflow configured properly for your system before attempting to run with real data.
+
+### The nf-core launch command
+
+Most _nf-core_ pipelines have a number of flags that need to be passed on the command line: some mandatory, some optional.
+To make it easier to launch pipelines, these parameters are described in a JSON file bundled with the pipeline.
+The `nf-core launch` command uses this to build an interactive command-line wizard which walks through the different options with descriptions of each, showing the default value and prompting for values.
+
+> _NB: This is an experimental feature - JSON file and rich descriptions of parameters is not yet available for all pipelines._
+
+Once all prompts have been answered, non-default values are saved to a `params.json` file which can be supplied to Nextflow to run the pipeline. Optionally, the nextflow command can be launched there and then.
+
+To use the launch feature, just specify the pipeline name:
+
+```bash
+nf-core launch <PIPELINE>
+```
+
+### Using nf-core pipelines offline
+
+Many of the techniques and resources described above require an active internet connection at run time - pipeline files, configuration profiles and software containers are all dynamically fetched when the pipeline is launched.
+This can be a problem for people using secure computing resources that do not have connections to the internet.
+
+To help with this, the `nf-core download` command automates the fetching of required files for running _nf-core_  pipelines offline.
+The command can download a specific release of a pipeline with `-r`/`--release` and fetch the singularity container if `--singularity` is passed (this needs Singularity to be installed).
+All files are saved to a single directory, ready to be transferred to the cluster where the pipeline will be executed.
+
+> _NB: At the time of writing (Sept 2019), functionality to download config files is not yet complete.
```

Comment:
is this already working now, no? I've tested with my pipeline and it works :)

Reply:
Yup, it works now! Still need to add functionality to customise the config to work with the downloaded singularity files though..